src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { Comment } from '../../react-app-env';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  return (
+    <>
+      {(comments.length)
+        ? (
+          comments.map(comment => (
+            <div className="CommentList">
+              <CommentInfo comment={comment} />
+            </div>
+          ))
+        ) : (
+          <b data-cy="NoCommentsMessage">No comments yet</b>
+        )}
```

Comment:
```suggestion
        )
        : <b data-cy="NoCommentsMessage">No comments yet</b>
        }
```

Reply:
Do you mean like this?
![image](https://user-images.githubusercontent.com/109978413/195901708-decf2ec4-a8c4-4b43-9895-a3a64fcb27b8.png)